CHANGELOG.md

```diff
@@ -53,6 +63,11 @@ aws lambda invoke --function-name $PREFIX-ReconciliationReportMigration $OUTFILE
 
 External tooling making use of `searchContext` in the `GET` `/granules/` endpoint will need to update to make use of standard pagination via `limit` and `page` scrolling, as `searchContext` is no longer supported/is an ES specific feature.
 
+#### CUMULUS-4006
+
+The async_operation_image property of the cumulus module should be [updated to pull the ECR image for cumuluss/async-operation:53](./packages/api/ecs/async-operation/README.md).
```

Comment:
This is great, but we should include it or a reference to it in the Migration notes.

Reply:
I think this is under the Migration notes, they start on line 48. Let me know if that's wrong, but I thought 3833 and 3967 were in the Migration notes, and added it after those.
Once this is approved, I plan on adding to the v20 release notes as well, as this change is under the v20 part of the changelog.

Reply:
Oops, sorry, I just misread the header indent - rendering it looks right.     Apologies for the confusion